tests/cupy_tests/linalg_tests/test_product.py

```diff
@@ -256,14 +258,16 @@ def test_transposed_tensordot(self, xp, dtype):
         b = testing.shaped_arange((4, 3, 2), xp, dtype).transpose(2, 0, 1)
         return xp.tensordot(a, b)
 
-    @testing.for_all_dtypes()
+    @testing.for_all_dtypes(excludes=(numpy.uint8, numpy.int8, numpy.uint16,
```

Comment:
Ah! I guess you mean that I need to "add a test code to check the behavior with those excluded types". I will add that later. Thank you!